eclipse-scout-core/src/desktop/outline/pages/Page.ts

```diff
@@ -126,13 +126,15 @@ export class Page extends TreeNode implements PageModel {
   }
 
   protected override _destroy() {
+    this.trigger('destroying');
```

Comment:
where are these events used?

Reply:
If a Scout JS page uses the `HybridManager` to set a Scout Classic form as its detail form or search form, this form needs to be closed when the page is destroyed. Therefore, the page listens on its own destroying-event to trigger a `form.close()`.